Sources/Datadog/RUMMonitor.swift

```diff
@@ -23,7 +23,7 @@ internal enum RUMHTTPMethod: String {
     }
 }
 
-internal enum RUMResourceKind {
+public enum RUMResourceKind {
```

Comment:
In #366 we merged a public enumeration for this: `RUMResourceEvent.Resource.ResourceType`, so we will be duplicating this information in public interface. Lets go with this `public enum RUMResourceKind` for `-alpha2`, but I will create JIRA to generate `RUMResourceEvent.Resource.ResourceType` as `RUMResourceKind` and update it before `1.5.0` GA. WDYT?

Reply:
it should be quick to replace `RUMResourceKind` with `ResourceType` but we should first discuss how to expose `RUMDataModels` to ObjC 🤔 until they are `@objc`, we will need to duplicate them manually in ObjC (which is not ideal)

so yes, let's keep it for now as it's done already and fix it before `1.5.0` 👍 
i'll leave a `TODO` here

Reply:
actually there is the same issue for `String -> RUMMethod` replacement: now i replaced `httpMethod: String` with `httpMethod: RUMMethod` but i also needed to create `DDRUMMethod` in ObjC (a duplication, basically)

i'd say let's fix both in this PR or let's leave both and see later, wdyt?

Reply:
replaced ✅